lib/Common/Memory/WaitForConcurrentCaller.h

```diff
@@ -0,0 +1,15 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include "Core/CommonTypedefs.h"
+
+enum WaitForConcurrentCaller : uint8
+{
+#define P(n) n,
+#include "WaitForConcurrentCallerInc.h"
```

Comment:
Does anyone else include this header file? If not, can it be collapsed into this header?

Reply:
yes, it gets included in the telemetry transmitter.